payroll/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Payroll",
-    "version": "16.0.1.0.0",
+    "version": "16.0.2.0.0",
```

Comment:
I think such high jump is not needed for this fix. I know that contributing guidelines expresses that it should be major when migrations scripts are needed, but it's referring to huge database model changes, not when we need to load an XML.